std/typecons.d

```diff
@@ -2522,6 +2527,15 @@ Returns:
         return _value;
     }
 
+    /// ditto
+    @property get(T)(T datum) @safe pure nothrow
```

Comment:
I tried, but I get a weird error and I don't understand exactly why : 
std/typecons.d(2531): Error: pure function 'std.typecons.Nullable!(VariantN!32LU).Nullable.get' cannot call impure destructor 'std.variant.VariantN!32LU.VariantN.~this'